python/libcugraph/libcugraph/load.py

```diff
@@ -45,19 +45,17 @@ def _load_wheel_installation(soname: str):
 def load_library():
     """Dynamically load libcugraph.so and its dependencies"""
     try:
-        # libraft must be loaded before libcugraph because libcugraph
-        # references its symbols
+        # These libraries  must be loaded before libcugraph because libcugraph
+        # references their symbols
+        import librmm
         import libraft
```

Comment:
Note: some of the logger PRs use alphabetical order here, while others use dependency tree order. Make this consistent.

Reply:
I tried to stick to dependency ordering.